core/src/main/java/com/novoda/espresso/AccessibilityServiceTestRule.java

```diff
@@ -0,0 +1,95 @@
+package com.novoda.espresso;
+
+import android.os.SystemClock;
+
+import org.junit.rules.TestRule;
+import org.junit.runner.Description;
+import org.junit.runners.model.Statement;
+
+class AccessibilityServiceTestRule implements TestRule {
```

Comment:
same as the old-school TalkBack one, but delegates to the Helper which is passed in, instead of a talkback only implementation.